packages/wrangler/src/routes.ts

```diff
@@ -24,7 +24,8 @@ export async function getWorkersDevSubdomain(
 			);
 
 			const wantsToRegister = await confirm(
-				"Would you like to register a workers.dev subdomain now?"
+				"Would you like to register a workers.dev subdomain now?",
+				{ fallbackValue: false }
```

Comment:
This is unrelated to the bundle controller—just a fix to make this default to false in non-interactive mode